pages/api/v1/activity-feed/subscribed-posts.ts

```
@@ -0,0 +1,335 @@
+	created_at?: Date;
+}
+
+async function handler(req: NextApiRequest, res: NextApiResponse) {
+	const network = String(req.headers['x-network']);
+	if (!network || !isValidNetwork(network)) return res.status(400).json({ message: messages.INVALID_NETWORK });
+
+	const token = getTokenFromReq(req);
+	if (!token) return res.status(400).json({ message: messages.INVALID_JWT });
+
+	const user = await authServiceInstance.GetUser(token);
+	if (!user || isNaN(user.id)) return res.status(403).json({ message: messages.UNAUTHORISED });
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)